### DISCUSSION TITLE
what happened to `antonmedv/expr`?

### DISCUSSION THREAD

**Discussion by Slahser** [argoproj/argo-workflows#11964] [Q&A] [answered]
this lib now 404...

**Comment by agilgur5** [comment]
Well that's worrisome. We interact with the maintainer relatively often too.

I was going to cc @antonmedv but it looks like the entire GitHub account is [deactivated](https://github.com/antonmedv) right now, not just the `expr` repo. The website is also not loading right now: https://expr.medv.io/. It may have been hosted on GH pages.

Hopefully the situation resolves itself, but in the interim, can see the docs at my fork: https://github.com/agilgur5/expr/blob/master/docs/Language-Definition.md.

I made a contribution earlier this week, so this is a very recent event, thanks for notifying us!

**Reply by agilgur5** [reply]
Looks like [the website](https://expr.medv.io/) is back up now. Account and repo still 404

**Reply by agilgur5** [reply]
Found the author's Twitter page from the website (and confirmed it was linked to [previously](https://web.archive.org/web/20220204200243/https://medv.io/)). Looks like GitHub [flagged the account](https://x.com/antonmedv/status/1711295286562189395?s=20) for some reason and hasn't responded since 😕

I retweeted for reach

**Comment by antonmedv** [comment, marked as answer]
My GitHub account got flagged. Trying to contact GitHub stuff, to figure out why. But still no response.

**Reply by antonmedv** [reply]
https://expr.medv.io now works.

Moved https://expr.medv.io to personal hosting from GitHub. I will eventually move to CloudFlare.

**Reply by agilgur5** [reply]
Back to normal now: https://github.com/antonmedv/expr.

Thanks for working through the issues and explaining the situation to everyone @antonmedv!
(See [Twitter thread](https://x.com/antonmedv/status/1711295286562189395?s=20) for details)

**Comment by terrytangyuan** [comment]
Thanks for bringing this to our attention. We are aware of this and are working with other maintainers to decide on an approach (other Argo projects also use it).

**Reply by annamedv** [reply]
We with @antonmedv working on resolving this issues right now. Also @antonmedv has a fork in a separate org here: https://github.com/expr-lang/expr

**Reply by terrytangyuan** [reply]
Thanks! We can probably switch to use that for now.

**Reply by agilgur5** [reply]
@annamedv could @antonmedv publicly confirm that this is an appropriate fork to use in the interim? Such as [in response on Twitter](https://x.com/agilgur5/status/1711427576462209405?s=20)?

Given the prevalence of supply chain attacks, a trusted source confirming is important.
I did confirm that the `git` SHAs on GitHub match at least: https://github.com/expr-lang/expr/commit/709c5dd55aa7cfc1b47fc67ee32ac15af6db967a is the same as https://github.com/agilgur5/expr/commit/709c5dd55aa7cfc1b47fc67ee32ac15af6db967a. Although it is a few commits behind the tip

**Reply by agilgur5** [reply]
@antonmedv [confirmed on Twitter](https://x.com/antonmedv/status/1711448876677280051?s=20) that this is a fork they created some time ago and is safe to use.
